scripts/migrate_hardcoded_questions.py

```
@@ -0,0 +1,209 @@
+#!/usr/bin/env python3
```

Ruff (0.12.2)

Shebang is present but file is not executable

(EXE001)